autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -1,28 +1,19 @@
 import * as React from 'react';
-import {
-  List, ListItemText, ListItem, Checkbox, ListItemIcon, Typography, ListSubheader,
-  Tooltip, Divider,
-} from '@material-ui/core';
-import HonorsIcon from '@material-ui/icons/School';
-import { useSelector, useDispatch } from 'react-redux';
-import Meeting, { MeetingType } from '../../../../../../types/Meeting';
-import Section from '../../../../../../types/Section';
-import { formatTime } from '../../../../../../utils/timeUtil';
+import { List, Typography } from '@material-ui/core';
+import { useSelector } from 'react-redux';
 import { SectionSelected } from '../../../../../../types/CourseCardOptions';
 import { RootState } from '../../../../../../redux/reducer';
 import * as styles from './SectionSelect.css';
-import { updateCourseCard } from '../../../../../../redux/actions/courseCards';
-import GradeDist from './GradeDist/GradeDist';
+import SectionInfo from './SectionInfo';
 
 interface SectionSelectProps {
   id: number;
 }
 
 const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
   const sections = useSelector<RootState, SectionSelected[]>(
-    (state) => state.courseCards[id].sections,
+    (state) => state.courseCards[id].sections, (left, right) => left === right,
```

Comment:
I thought the default equality function checked reference equality, why did you explicitly do it here?

Reply:
Nice catch. I think I put that in while debugging to see if it would change anything, and understandably, it didn't. Removed in latest commit.